doc/admin/installation.md

```diff
@@ -176,8 +172,7 @@ On Mac, please make sure that the Docker settings under
 To build the Docker image directly from GitHub, run the following command:
 
 ```bash
-$ curl -L https://raw.githubusercontent.com/daos-stack/daos/master/utils/docker/Dockerfile.centos.7 | \
-        docker build --no-cache -t daos -
+$ docker build https://github.com/daos-stack/daos.git#master:utils/docker/Dockerfile.centos.7
```

Comment:
I'll assume this is tested and works.

Reply:
Yes, that's was we actually used to do, but we have to move to the curl command line to address some issue with the sl repository. Now that everything is integrated into the DAOS repo, we could switch back to this method.